BaseballMitt.cpp

```diff
@@ -5,10 +5,22 @@ BaseballMitt::BaseballMitt(std::string a_color) :
     color(std::move(a_color))
 {}
 
-void BaseballMitt::print_type() const {
-    std::cout << "BaseballMitt Type is " << this->type << std::endl;
+int BaseballMitt::get_type() const {
+    return type;
 }
 
 void BaseballMitt::set_type(int type) {
     this->type = type;
 }
+
+const std::string & BaseballMitt::get_color() const {
+    return color;
+}
+
+void BaseballMitt::set_color(std::string color) {
+    this->color = std::move(color);
+}
```

Comment:
This style of explicitly using `this->` is highly unusual. Basically, nobody does it this way, at least not that I've seen.

A common style I've seen is that data members have a `_` suffix (`color_`) or a `m_` prefix (`m_color`), and then this function would be written like the following:
```
void BaseballMitt::set_color(std::string color) {
    color_ = std::move(color);
}
```